### ISSUE TITLE
Revert <head> control Blazor guidance

### ISSUE THREAD

**Issue by guardrex** [dotnet/AspNetCore.Docs#20319]
**NOTE**: There is deactivated `Link` component content in the *Class Libraries* topic to update/reactivate for 6.0.

When components for influencing `<head>` elements come back, revert changes made on  #20318.

TOC *Built-in components* entries

```
- name: Link
  href: blazor/fundamentals/additional-scenarios.md#influence-html-head-tag-elements
- name: Meta
  href: blazor/fundamentals/additional-scenarios.md#influence-html-head-tag-elements
- name: Title
  href: blazor/fundamentals/additional-scenarios.md#influence-html-head-tag-elements
```

Release notes ...

```
### Influence the HTML head in Blazor apps

Use the new `Title`, `Link`, and `Meta` components to programmatically set the title of a page and dynamically add links and meta tags to the HTML head in a Blazor app. For more information, see <xref:blazor/fundamentals/additional-scenarios#influence-html-head-tag-elements>.
```

Also, update the following line ...

> \*This section applies to the upcoming ASP.NET Core 5.0 release of Blazor WebAssembly and Blazor Server.\*

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: fc56d254-09c5-993b-2559-76af9a3f97ed
* Version Independent ID: a0bb183f-7e4d-ba39-c94c-cf60e951c7a9
* Content: [ASP.NET Core Blazor hosting model configuration](https://docs.microsoft.com/en-us/aspnet/core/blazor/fundamentals/additional-scenarios?view=aspnetcore-5.0)
* Content Source: [aspnetcore/blazor/fundamentals/additional-scenarios.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/blazor/fundamentals/additional-scenarios.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-blazor**
* GitHub Login: @guardrex
* Microsoft Alias: **riande**

**Comment by guardrex** [comment]
@captainsafia ... Will the work for bringing back `<head>` control in Blazor apps be done for the 6.0 release? I just want to get the labeling right for this docs issue.

**Comment by captainsafia** [comment]
@guardrex It's on the roadmap for 6.0 but prioritized pretty low. I would tentatively put this on-hold until we confirm we are doing this for 6.0.

**Comment by guardrex** [comment]
Note to self:

https://github.com/dotnet/aspnetcore/blob/main/src/ProjectTemplates/Web.ProjectTemplates/content/BlazorServerWeb-CSharp/Pages/_Layout.cshtml#L15

**Comment by guardrex** [comment]
This was resolved by #22931, #22929, and #22931.
